bindings/python/gnucash_core.py

```diff
@@ -426,16 +428,18 @@ def __args_to_instance(args):
         elif len(args) == 1:
             arg = args[0]
             if isinstance(arg, int):
-                return gnc_numeric_create(arg ,1)
+                return gnc_numeric_create(arg, 1)
             elif isinstance(arg, float):
                 return double_to_gnc_numeric(arg, GNC_DENOM_AUTO, GNC_HOW_DENOM_FIXED | GNC_HOW_RND_NEVER)
```

Comment:
Just tried: In [22]: print(gnucash.GncNumeric(2.2))                                                                                                                                                       
8593750000000001/3906250000000000

That is not what I would have expected.

Reply:
Yup. Binary decimals like double and float are poison. This was driven home for me a few years ago when we had complaints of Finance::Quote prices turning into weird fractions. We fixed it by using the Guile equivalent of Decimal for importing the prices.